templates/CRM/Event/Form/SelfSvcUpdate.tpl

```diff
@@ -31,7 +31,7 @@
         <td class="right nowrap crm-paticipant-participant_fee_amount">{$details.fee_amount}</td>
       {/if}
       <td>{$details.register_date|truncate:10:''|crmDate}</td>
-      <td>{$details.status}</td><td class="crm-participant-participant_role">{$details.role}</td>
+      <td>{$details.statuslabel}</td><td class="crm-participant-participant_role">{$details.rolelabel}</td>
```

Comment:
This looks good just these should be escaped since the label is arbitrary user input, e.g. `{$details.statuslabel|escape}`

Not a blocker but also I notice SelfSvcTransfer.tpl has the same problem. It seems to get the data from the same php so making the same change in that tpl would solve that too.